.eslintignore

```diff
@@ -0,0 +1,2 @@
+/cypress/e2e/*
```

Comment:
You don't need to ignore these files if you have your eslint config correct

Reply:
Fixed